WordPress/Classes/Services/CommentServiceRemoteCoreRESTAPI.swift

```diff
@@ -20,11 +20,16 @@ final class CommentServiceRemoteCoreRESTAPI: NSObject, CommentServiceRemote {
     func getCommentsWithMaximumCount(_ maximumComments: Int, options: [AnyHashable: Any]? = [:], success: (([Any]?) -> Void)?, failure: (((any Error)?) -> Void)?) {
         Task { @MainActor in
             do {
-                let sequence = await client.api.comments.sequenceWithEditContext(params: .init(options: options))
+                let params = CommentListParams(options: options, perPage: min(100, UInt32(maximumComments)))
```

Comment:
The endpoint returns 10 comments by default, which is too few. I hard-coded 100 max comments per request.